src/unit/test_help.h

```diff
@@ -28,6 +28,8 @@
 #define KBLUE  "\33[34m"
 #define KRESET "\33[0m"
 
+#define TEST(name) printf("test — %s\n", name);
```

Comment:
As per the new test framework, probably we can stop using this `TEST` which was originally  used for printing different test case names.

Reply:
Yes, let's stop using TEST, which really should have been individual tests.